server/config/keys.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-	mongoURI: "mongodb+srv://dbAdmin:teamcactus@hatch-ptlkp.mongodb.net/test"
+	mongoURI: "mongodb+srv://dbAdmin:teamcactus@hatch-ptlkp.mongodb.net/test",
```

Comment:
For local development, you should use a local server no? 
We'd probably want to make these environment variables (in .env) and then gitignore those files